### PR TITLE
3405 Enhance doc-string of MIL model for data shape

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #3405 .

### Description
This PR enhanced doc-string of MIL model to help users understand the expected data shape, like #3405 .

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Review by yiheng-wang-nv — APPROVED** [review]

File: monai/networks/nets/milmodel.py [hunk 1]
```diff
@@ -11,6 +11,10 @@
 class MILModel(nn.Module):
     """
     Multiple Instance Learning (MIL) model, with a backbone classification model.
+    Currently, it only works for 2D images, typical use case is for classification of the
```
Comment (wyli) [inline monai/networks/nets/milmodel.py:14]: looks like this model will be 2D only, the name `MILModel` is too generic and confusing cc@myron

**Pull request merged**